### DISCUSSION TITLE
Release v0.0.0-test RSS Feed notification

### DISCUSSION THREAD

**Discussion by vaprzemek** [grafana/grafana#63271] [General]
Hello
In the last few weeks I have been getting notifications from the RSS feed about new tags, which looks like you are testing/adjusting something in your release process.
![image](https://user-images.githubusercontent.com/44766208/218131633-46090638-4bd6-482a-abe6-27f4e361d2df.png)

Since GitHub has put new tags and new releases under one RSS feed (https://github.com/grafana/grafana/releases.atom), I am not able to mute it (also, I use a simple RSS reader as a slack application instead of a dedicated RSS reader).

Would you consider moving your tests to the other repository instead of using this one?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This question seems like something better for the [community forum](https://community.grafana.com/) if you are still having this problem.
